DATA/EQUIPMENT/shield_equip.ini

```diff
@@ -3965,7 +3965,7 @@ ids_info = 461030
 ; Mass: $mass
 ; Integrity: $hitpoints
 ; 
-; Developed by Aegeria Technologies. it is a basic graviton shield mostly sold with new ships. Graviton shields have high protection against laser and photon weapons but are weak against particle and plasma projectiles.
+; Developed by Aegeria Technologies. it is a basic graviton shield mostly sold with new ships. Graviton Shields are good against Laser, Tachyon and Photon peojectiles while it is weak against Plasma, Particle and Neutron guns

```

Comment:
The "peojectiles" mistake is on many many entries here, they still need to be fixed.